<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>200</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="testChooseFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QWidget" name="datasetLayout" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <italic>true</italic>
           <bold>false</bold>
          </font>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="datasetLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>18</pointsize>
              <italic>true</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Тесты: </string>
            </property>
            <property name="textFormat">
             <enum>Qt::TextFormat::PlainText</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="datasetComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>Неправильные глаголы 1</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="separator1">
         <property name="frameShape">
          <enum>QFrame::Shape::HLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="testTypeLayout" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="difficultyLayout" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>20</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="difficultyLabel">
               <property name="font">
                <font>
                 <pointsize>18</pointsize>
                 <italic>true</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Сложность </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="difficultyComboBox">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>1000</width>
                 <height>1000</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>16</pointsize>
                 <italic>true</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>1 уровень (исчезает 1-ая форма)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2 уровень (исчезает 2-ая форма)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3 уровень (исчезает 3-я форма)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4 уровень (написать 2 и 3 форму)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5 уровень (написать все формы)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>6 уровень (всё, кроме 3 формы)</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="separator2">
         <property name="frameShape">
          <enum>QFrame::Shape::HLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="horizontalWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>15</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="startFlowButton">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <italic>true</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Начать обучение</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="startTestButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <italic>true</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Начать тест</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>0</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QMenu" name="mainMenu">
    <property name="title">
     <string>Главное</string>
    </property>
    <addaction name="showAboutProgram"/>
    <addaction name="openMainSettings"/>
    <addaction name="exitProgram"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Наборы данных</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_3"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Таблицы результатов</string>
    </property>
    <addaction name="action_7"/>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
    <addaction name="action_8"/>
   </widget>
   <widget class="QMenu" name="debugMenu">
    <property name="title">
     <string>Отладка</string>
    </property>
    <addaction name="showTestResultDialog"/>
    <addaction name="requestDatasetUpdateAction"/>
    <addaction name="showDatasetContentAction"/>
   </widget>
   <addaction name="mainMenu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="debugMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <action name="action">
   <property name="text">
    <string>Редактирование ресурсов</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Перезагрузить наборы</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Редактировать наборы</string>
   </property>
  </action>
  <action name="showAboutProgram">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="openMainSettings">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="exitProgram">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Показать</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Очистить</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Выгрузить</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="showTestResultDialog">
   <property name="text">
    <string>Показать диалог результата теста</string>
   </property>
  </action>
  <action name="showDatasetContentAction">
   <property name="text">
    <string>Запросить строку с содержанием датасета</string>
   </property>
  </action>
  <action name="requestDatasetUpdateAction">
   <property name="text">
    <string>Запросить обновление списка датасетов</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
